src/main/java/bio/terra/tanagra/app/controller/UnauthenticatedApiController.java

```diff
@@ -0,0 +1,30 @@
+package bio.terra.tanagra.app.controller;
+
+import bio.terra.tanagra.generated.controller.UnauthenticatedApi;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.Optional;
+import javax.servlet.http.HttpServletRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+
+/** Spring controller for the unauthenticated API methods. */
+@Controller
+public class UnauthenticatedApiController implements UnauthenticatedApi {
+  @Override
+  public ResponseEntity<Void> serviceStatus() {
+    return new ResponseEntity<>(HttpStatus.OK);
+  }
+
+  /** Required if using Swagger-CodeGen, but actually we don't need this. */
```

Comment:
I'm not familiar with this `ObjectMapper` pattern - is this required? Or is this an exception to the normal controller pattern here?

Reply:
Swagger-codegen generated Class has that two methods